validation/jinja2/validation/segment_details.html

```diff
@@ -36,19 +36,22 @@
         <button class="button button--success button--back" onclick='goBack()'>Back</button>
     </div>
 
+    <p>This entry was last edited by {{ history[0].modifier }} on {{ history[0].history_date.strftime('%Y-%m-%d') }}</p>
```

Comment:
In case the `history` list is empty, this `<p>` should be wrapped in a conditional.

(I don’t know how often `history` might be empty, but in dev mode, `loaddata` seems to create entries without any history.)